src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1648,7 +1642,7 @@ protected virtual void ValidatePropertyType(BindingDiagnosticBag diagnostics)
             {
                 diagnostics.Add(ErrorCode.ERR_FieldCantBeRefAny, TypeLocation, type);
             }
-            else if (this.IsAutoPropertyWithGetAccessor && type.IsRefLikeType && (this.IsStatic || !this.ContainingType.IsRefLikeType))
+            else if (this.IsAutoProperty && type.IsRefLikeType && (this.IsStatic || !this.ContainingType.IsRefLikeType))
```

Comment:
>this.IsAutoProperty [](http://example.com/codeflow?start=21&length=19)

The motivation behind this change is not clear to me. Could you please elaborate?